api/references/when-clause-contexts.md

```diff
@@ -288,11 +288,11 @@ View identifiers:
 
 You can have a when clause that checks if a specific [View Container](/api/ux-guidelines/views#view-containers) is visible
 
-Context key     | True when
---------------- | ------------
-activeViewlet   | True when view container is visible in the sidebar.<br>Example: `"activeViewlet == 'workbench.view.explorer'"`
-activePanel     | True when view container is visible in the panel.<br>Example: `"activePanel == 'workbench.panel.output'"`
-activeAuxiliary | True when view container is visible in the secondary sidebar.<br>Example: `"activeAuxiliary == 'workbench.view.debug'"`
+Context name      | True when
```

Comment:
@muzimuzhi is there any reason we changes `Context key` to `Context name`?

Reply:
Because on the same page, "Context name" is used in the two other similar places, so I think "Context key" was an exception:
https://github.com/microsoft/vscode-docs/blob/d239f68f168aad63d57202702898d623999d6649/api/references/when-clause-contexts.md?plain=1#L142-L143
https://github.com/microsoft/vscode-docs/blob/d239f68f168aad63d57202702898d623999d6649/api/references/when-clause-contexts.md?plain=1#L260-L261
https://github.com/microsoft/vscode-docs/blob/d239f68f168aad63d57202702898d623999d6649/api/references/when-clause-contexts.md?plain=1#L291-L292

Reply:
Okay, thanks for clarification